### PR TITLE
Update to Node to v20.x

### PR DESCRIPTION
**Pull request by sapiderman**
# Monika Pull Request (PR)

## What feature/issue does this PR add

1. Implements #1234 
2. Updating node to v20.x

## How did you implement / how did you fix it

1. Update all pipelines
2. Regenerate package.json

## How to test

1. Pushing the branch here first to test the pipelines. All pipelines should be green
2. `npm run test` should all work without errors

## Screenshot
image
(updated screenshot)

### CODE REVIEW

File: package.json
Comment (dennypradipta) [inline package.json:1]: In the 'engines' key, you need to update the node to be 20.x too

**Review by dennypradipta — CHANGES_REQUESTED** [review]
A little request changes.

File: package.json
Comment (sapiderman) [reply package.json:1]: got it...good catch thanks!

**Review by haricnugraha — APPROVED** [review]

**Review by dennypradipta — APPROVED** [review]
Nicely done.

**Pull request merged**